### DISCUSSION TITLE
AI在藝術上的應用

### DISCUSSION THREAD

**Discussion by 32844583** [1101-GS3073-C/Python-Community-GS3073-C#536] [General]
### Fast style transfer
由 Justin Johnson et al. (2016)在論文 Perceptual Losses for Real-Time Style Transfer and Super-Resolution 提出，主要特點在於將Gatys 的風格遷移模型從更新圖片的方式轉為以單一的風格網路學習，從而節省推論時間，不需要再對每個圖片對做訓練。
Github:https://github.com/pytorch/examples/tree/main/fast_neural_style
Gatys介紹:https://medium.com/ai-academy-taiwan/類神經影像藝術風格轉換系列筆記-基礎-e5a02d07c5c0

**Comment by BingRong1217** [comment]
AI可以應用在藝術上超酷的!!

**Comment by limitx0** [comment]
影像風格轉換好酷
沒想到AI做風格轉換也做得不錯

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by boson13579** [comment]
之前有看過一篇報導，內容提到專家們應用AI修復因為氧化掉色、殘缺的古畫和因為戰爭或其他人為破壞而難以看清全貌的石雕。藉由AI的幫助，我們才能透過被復原的藝術品了解古人的風俗習慣及工藝技術。
